webapp/config/load_db_secrets.php

```diff
@@ -34,7 +34,7 @@ function get_db_url(): string
         break;
     }
 
-    return sprintf('mysql://%s:%s@%s:%d/%s?serverVersion=5.7.0', $user, $pass, $host, $port ?? 3306, $db);
+    return sprintf('mysql://%s:%s@%s:%d/%s?serverVersion=mariadb-10.7.3', $user, $pass, $host, $port ?? 3306, $db);
```

Comment:
Do we want this?

Reply:
It was for me to check if that would make it easier to generate doctrine:migrations:diff.

But I still needed to manually fix the migrations so the commit message already reminds me to later remove it.

Reply:
I have a similar thing on my Linux machine but think I didn't have it on my mac with Docker (even after adding stuff to `.env.local`). We might want to investigate later.